src/app/utils.py

```diff
@@ -518,26 +518,22 @@ def check_spdx_license(licenseText):
     spdxLicenseTexts = r.mget(spdxLicenseIds)
     licenseData = dict(list(zip(spdxLicenseIds, spdxLicenseTexts)))
     matches = get_close_matches(licenseText, licenseData)
-
     if not matches:
         matchedLicenseIds = None
         matchType = 'No match'
-    
-    elif 1.0 in list(matches.values()) or all(0.99 < score for score in list(matches.values())):
-        matchedLicenseIds = list(matches.keys())
-        matchType = 'Perfect match'
-    
     else:
-        for licenseID in matches:
-            listedLicense = getListedLicense(licenseID)
-            isTextStandard = checkTextStandardLicense(listedLicense, licenseText)
-            if not isTextStandard:
-                matchedLicenseIds = licenseID
-                matchType = 'Standard License match'
-            else:
-                matchedLicenseIds = max(matches, key=matches.get)
-                matchType = 'Close match'
-    return matchedLicenseIds, matchType
+        matchedLicenseIds = max(matches, key=matches.get)
+        if matches[matchedLicenseIds] > 0.99:
+            matchType = 'Perfect match'
+        else:
+            matchType = 'Close match'
+            for licenseID in matches:
```

Comment:
We are looping over all matched licenses list but returning only the last standard license in list or the one which close match. @goneall Should we return all the matched licenses or the license with the max match percent? We need some way to show match percentages vs license and let users choose?

Reply:
We could probably improve this.  Considerations would be the score and also if the SPDX listed license is deprecated - there are a lot of duplicate license texts where the license is deprecated.  If we match a deprecated and a non-deprecated license, we should only return the non-deprecated.

We may want to open a separate PR and/or Issue since this would be a pretty sizeable change.